libvast/test/system/meta_index.cpp

```diff
@@ -268,6 +285,35 @@ TEST(attribute extractor - type) {
   CHECK_EQUAL(lookup("#type !~ /x/"), ids);
 }
 
+TEST(attribute extractor - age) {
```

Comment:
Can you add a comment that half the test data was set to import age 1975 and the other half to 2015 in the test setup?